compile/compile.go

```diff
@@ -418,7 +418,7 @@ func (c *Compiler) optimize(ctx context.Context) error {
 	return nil
 }
 
-func (c *Compiler) compilePlan(ctx context.Context) error {
+func (c *Compiler) compilePlan(_ context.Context) error {
```

Comment:
[nit]
```suggestion
func (c *Compiler) compilePlan(context.Context) error {
```

Reply:
Sure... but can't we have linter rules for things like this? 😅

Reply:
Done :)